src/item.c

```diff
@@ -880,6 +880,7 @@ const u8 *ItemId_GetName(u16 itemId)
     return gItems[SanitizeItemId(itemId)].name;
 }
 
+// Unused
 u16 ItemId_GetId(u16 itemId)
```

Comment:
But upstream didn't remove this function. They just removed the header declaration for some reason. You should probably put it back, since the replacement is not the same: the item ID that gets passed in is passed through `SanatizeItemId` here, which will quash glitch or invalid item values.

Reply:
While it's true that it sanitizes the id, implementing this properly would mean that we would need to go to every instance where an item id is used and add `ItemId_GetId` to it. I'm not sure how worth it is it for a function that wasn't used in vanilla and that nobody realistically would use, specially in our case as we removed unused items from it.